test/typings-test.ts

```diff
@@ -0,0 +1,128 @@
+import { alpha2ToAlpha3, alpha2ToNumeric, alpha3ToAlpha2, alpha3ToNumeric, toAlpha2, toAlpha3 } from "..";
```

Comment:
The goal with these tests are to just make a program that doesn't contain type errors, but ensures that inference is happening properly.